package/piksi_leds/src/main.c

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright (C) 2017 Swift Navigation Inc.
+ * Contact: Gareth McMullin <gareth@swiftnav.com>
+ *
+ * This source is subject to the license found in the file 'LICENSE' which must
+ * be be distributed together with this source. All other rights reserved.
+ *
+ * THIS CODE AND INFORMATION IS PROVIDED "AS IS" WITHOUT WARRANTY OF ANY KIND,
+ * EITHER EXPRESSED OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE IMPLIED
+ * WARRANTIES OF MERCHANTABILITY AND/OR FITNESS FOR A PARTICULAR PURPOSE.
+ */
+
+#include <libpiksi/sbp_zmq_pubsub.h>
+#include <libpiksi/logging.h>
+#include <libpiksi/util.h>
+
+#include "firmware_state.h"
+#include "manage_led.h"
+
+#define PROGRAM_NAME "piksi_leds"
+
+#define PUB_ENDPOINT ">tcp://localhost:43031"
```

Comment:
As someone new to the company... these seemingly arbitrary port numbers are hard to grok.  What is the port pair 43031/43030 for?  Can we name the #define such that it captures it's intended usage?